### PR TITLE
Implement tasks homework_03

### PR DESCRIPTION
**Pull request by AnastasiyaGavrilik**

### CODE REVIEW

File: HW03/Anastasiya_Gavrilik/lib/app/homeworks.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module App
+  class Homeworks
```
Comment (vladhilko) [inline HW03/Anastasiya_Gavrilik/lib/app/homeworks.rb:2]: Maybe?
```suggestion
  class Homework
```

File: HW03/Anastasiya_Gavrilik/lib/app/student.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module App
+  class Student < Human
+    attr_reader :homeworks
+    def initialize(name:, surname:)
+      super
+      @homeworks = []
+    end
+
+    def submit_homework!
+      @homeworks.last.author.notify(self)
```
Comment (violettaMi) [inline HW03/Anastasiya_Gavrilik/lib/app/student.rb:10]: 👍

File: HW03/Anastasiya_Gavrilik/lib/app/student.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    end
+
+    def get_and_do_homework(mentor)
+      @homeworks <<  mentor.homeworks.find(&new_homework)
+    end
+
+    def new_homework
+      ->(homework) { !@homeworks.include?(homework.title) }
```
Comment (violettaMi) [inline HW03/Anastasiya_Gavrilik/lib/app/student.rb:18]: 👍

File: HW03/Anastasiya_Gavrilik/lib/app/notifications.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+module App
+  class Notifications
```
Comment (vladhilko) [inline HW03/Anastasiya_Gavrilik/lib/app/notifications.rb:2]: Where do you use this class? 🤔

**Pull request merged**